db/migrate/20230316050422_image.rb

```diff
@@ -0,0 +1,4 @@
+class Image < ActiveRecord::Migration[7.0]
+  def change
+  end
+end
```

Comment:
Seems like we're missing the content of the migration.

Reply:
content added